Feature/update docs (#280)
rl-institut/oemof-B3: docs/how_to_customize_oemof-B3.rst

.. _how_to_customize_oemof-B3_label:

~~~~~~~~~~~~~~~~~~~~~~~~~
How to customize oemof-B3
~~~~~~~~~~~~~~~~~~~~~~~~~

.. contents:: `Contents`
    :depth: 1
    :local:
    :backlinks: top

> To use oemof-B3, you may want to modify some files, depending on the setup of your Energy System.

Reviewer: ```suggestion
To use oemof-B3, you may want to modify some files, depending on what you want to model.
```
